src/io/flutter/utils/FileUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+import com.intellij.openapi.util.SystemInfoRt;
+import com.intellij.util.SystemProperties;
+
+public class FileUtils {
+  private static final String IDE_VENDOR_NAME = System.getProperty("idea.vendor.name", "JetBrains");
```

Comment:
I'm not sure JetBrains approves of outsiders (like us) modifying their internal data. If we make this "Google" then it can be shared with all IDEs, and the file will be only downloaded once.